src/__experimental__/components/grouped-character/grouped-character.stories.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Store, State } from '@sambego/storybook-state';
+import { text, object } from '@storybook/addon-knobs';
+import GroupedCharacter from './grouped-character.component';
+
+const groupedCharacterStore = new Store({
+  value: ''
+});
+
+const onChange = (ev) => { groupedCharacterStore.set({ value: ev.target.value }); };
+
+storiesOf('Experimental/GroupedCharacter', module)
+  .addParameters({
+    info: { propTablesExclude: [State] }
+  })
+  .add('Basic',
```

Comment:
if we want to stick to the naming convention we already have in storybook and keep some consistency, it may be better to change the name from 'basic' to 'default'